draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -800,19 +800,208 @@ support SVG rendering. The object contains the following properties:
+      assets to minimize correlation.
+
+##### Authoring Guidance (Non-Normative) {#logo-guidance}
+
+- Provide an `image/svg+xml` variant plus raster fallbacks at `scale` `2` and
+  `3` where feasible.
+- Use transparent backgrounds for raster logos unless your brand requires a
+  specific solid backdrop.
+- Keep file sizes small (each raster variant ≤ 200 KB; total ≤ 800 KB).
+- Avoid embedding text that duplicates UI labels; keep the mark clear at small
+  sizes.
+
+##### Example {#logo-example}
+
+The following shows a non-normative, reduced example of a logo object:
 
-- `uri`: A URI pointing to the logo image. This property is REQUIRED.
-- `uri#integrity`: An "integrity metadata" string as described in
-  (#document-integrity). This property is OPTIONAL.
-- `alt_text`: A string containing alternative text for the logo image. This
-  property is OPTIONAL.
+```json
+{
+  "logo": {
+    "preferred_max_height": 24,
+    "background_requirement": "transparent",
+    "variants": [
+      {
+        "appearance": "any",
+        "format": "image/svg+xml",
+        "uri": "https://cdn.example.org/brand/logo.svg",
+        "uri#integrity": "sha256-BASE64URL(...)"
+      },
+      {
+        "appearance": "dark",
+        "scale": 3,
+        "format": "image/png",
+        "size": { "width": 360, "height": 72 },
+        "uri": "https://cdn.example.org/brand/logo-dark@3x.png",
+        "uri#integrity": "sha256-BASE64URL(...)",
+        "max_age": 2592000
+      }
+    ],
+    "alt_text": "Example University"
+  }
+}
+```
+
+#### Background Image {#background-image}
+
+The `background_image` property is an object describing a decorative background image for the credential surface. Wallets use it to improve visual presentation and MUST NOT derive semantics from it.
+
+The object contains the following properties:
+
+- `variants`: An array of objects, each describing a renderable asset variant. If omitted, the background image is not used. See (#background-image-variant). This property is REQUIRED if `background_image` is present.
+- `safe_area`: An object defining insets where wallet UI SHOULD avoid placing overlaid text or controls. The object contains integer properties `top`, `right`, `bottom`, `left` that denote logical pixels at a scale factor of 1. This property is OPTIONAL.
+- `content_contrast`: A string that guides text color selection over the image. Allowed values are `light-on-dark`, `dark-on-light`, or `auto`. Wallets MAY ignore this value if they compute sufficient contrast. This property is OPTIONAL.
+
+##### Background Image Variant {#background-image-variant}
+
+Each variant object has:
+
+- `uri`: A HTTPS URI to the image asset. REQUIRED.
+- `uri#integrity`: Integrity metadata as described in (#document-integrity). Wallets SHOULD verify when present. OPTIONAL.
+- `format`: Media type (e.g., `image/png`, `image/jpeg`, `image/webp`, `image/svg+xml`). OPTIONAL.
```

Comment:
wanting to make https://github.com/oauth-wg/oauth-sd-jwt-vc/pull/335/files#r2387686645 again and then worried about duplication